src/coreclr/jit/disasm.cpp

```diff
@@ -1418,12 +1431,12 @@ void DisAssembler::disSetMethod(size_t addr, CORINFO_METHOD_HANDLE methHnd)
     if (disComp->eeGetHelperNum(methHnd))
     {
         DISASM_DUMP("Helper function: %p => %p\n", addr, methHnd);
-        GetHelperAddrToMethodHandleMap()->Set(addr, methHnd);
+        GetHelperAddrToMethodHandleMap()->Set(addr, methHnd, AddrToMethodHandleMap::SetKind::Overwrite);
```

Comment:
what are the scenarios with this change that we will have to overwrite the existing entry in a map?

Reply:
I think if there are multiple calls to the same function or helper in a method, it will be overwritten (with the same value).

It turns out we currently don't use this in the coredistools-based disassembler.